bash/manpage

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env bash 
-
-pandoc man/MANPAGE.md -s -t man -o man/pi.1
+#!/usr/bin/env bash 
+
+pandoc man/MANPAGE.md -s -t man -o man/pi.1
```

Comment:
The Clap team is about to release `clap_man`, a crate that generates the manpage for us: https://github.com/clap-rs/clap/tree/924262b329177cf569cb30e279809a16e3b5a587/clap_man, we can probably get rid of this at that time